apps/site/pages/en/learn/test-runner/mocking.md

```diff
@@ -149,20 +149,19 @@ This leverages [`mock`](https://nodejs.org/api/test.html#class-mocktracker) from
 import assert from 'node:assert/strict';
 import { before, describe, it, mock } from 'node:test';
 
-
-describe('foo', { concurrency: true }, () => {
+describe('foo',  concurrency: true }, () => {
```

Comment:
Syntax doesn't look right here, where's the opening `{` for the object?

Reply:
Ops mb, this was a test and I forgot to rollback. 

![image](https://github.com/user-attachments/assets/6a5804ae-9bb2-424d-9e70-88602010ffc9)